### DISCUSSION TITLE
Help needed: Implementing "Sign in with Google" in .NET MAUI

### DISCUSSION THREAD

**Discussion by NazrinAli12** [CommunityToolkit/Maui#2414] [General]
Hi CommunityToolkit,

I am currently learning .NET MAUI and trying to implement the "Sign in with Google" feature. However, I am facing some difficulties, as I need to do this without using Microsoft Azure, as per my teacher's recommendation.

Could you please guide me on how to implement this feature or share any resources or sample code that could help? I would greatly appreciate your support!

Thank you in advance for your time and help.

Best regards,
Nazrin

**Comment by VladislavAntonyuk** [comment]
https://www.linkedin.com/posts/umesh-kamble-7b459515_dotnetmaui-communitytoolkit-dotnet-ugcPost-7278730292041777152-07Cq?utm_source=share&utm_medium=member_ios
